Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -56,14 +56,9 @@ if(APPLE)
 	)
 
 	# Copy resources in the bundle
-	file(GLOB_RECURSE resources RELATIVE "${CMAKE_SOURCE_DIR}/Data" "${CMAKE_SOURCE_DIR}/Data/Sys/*")
-	foreach(res ${resources})
-		target_sources(${DOLPHINQT2_BINARY} PRIVATE "${CMAKE_SOURCE_DIR}/Data/${res}")
-		get_filename_component(resdir "${res}" DIRECTORY)
-		set_source_files_properties("${CMAKE_SOURCE_DIR}/Data/${res}" PROPERTIES
-			MACOSX_PACKAGE_LOCATION "Resources/${resdir}")
-		source_group("Resources" FILES "${CMAKE_SOURCE_DIR}/Data/${res}")
-	endforeach()
+	include(DolphinCopyDirectory)
+	dolphin_copy_directory(dolphinqt2_copy_sys ${CMAKE_SOURCE_DIR}/Data/Sys ${BUNDLE_PATH}/Contents/Resources/Sys)
```

Comment:
Doesn't work, I think generator expressions aren't allowed as OUTPUT in add_custom_command